project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
   val romeVersion = "1.0"
   val jerseyVersion = "1.19.4"
   val playJsonVersion = "2.6.3"
```

Comment:
From memory the extensions lib pulled in its own version of play json. Maybe we should align and bump this too?